source/includes/options-mongod.yaml

```diff
@@ -1614,31 +1614,31 @@ description: |
 
   - ``none``
 
-  - ``snappy``
+  - :term:`snappy`
 
-  - ``zlib``
+  - :term:`zlib`
 ---
 program: mongod
-name: wiredTigerCollectionBlockCompressor
+name: wiredTigerIndexBlockCompressor
```

Comment:
we whitelist prefix compression for indexes and block compression for collections. this change seems odd.